### PR TITLE
refactor(keybindings): Use bubble tea keybindings

### PR DESCRIPTION
**Pull request by Vedu1996**
Closes https://github.com/hedhyw/json-log-viewer/issues/43

### CODE REVIEW

**Comment by Vedu1996** [comment]
Hey @hedhyw , please have a look and let me know what you think

**Comment by Vedu1996** [comment]
Fixed the PR title and lint issues

File: internal/app/stateviewrow.go [hunk 1]
```diff
@@ -56,11 +61,11 @@ func (s StateViewRow) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
...
 		return s.handleErrorOccuredMsg(msg)
-	case events.BackKeyClickedMsg:
-		return s.previousState.withApplication(s.Application)
-	case events.EnterKeyClickedMsg:
-		return s.previousState.withApplication(s.Application)
 	case tea.KeyMsg:
+		switch {
+		case key.Matches(msg, s.keys.Back), key.Matches(msg, s.keys.ToggleView):
+			return s.previousState.withApplication(s.Application)
+		}
```
Comment (hedhyw) [inline internal/app/stateviewrow.go:68]:
```suggestion
		if key.Matches(msg, s.keys.Back) || key.Matches(msg, s.keys.ToggleView) {
			return s.previousState.withApplication(s.Application)
		}

```

File: internal/pkg/events/events.go [hunk 2]
```diff
@@ -54,22 +42,25 @@ func ViewRowsReloadRequested() tea.Msg {
...
 func FilterKeyClicked() tea.Msg {
-	return FilterKeyClickedMsg{}
+	return tea.KeyMsg{
+		Type:  tea.KeyRunes,
+		Runes: []rune{'f'},
+	}
 }
 
-// BackKeyClicked implements tea.Cmd. It creates BackKeyClickedMsg.
+// BackKeyClicked implements tea.Cmd.  It creates a message indicating 'Esc' has been clicked.
```
Comment (hedhyw) [inline internal/pkg/events/events.go:63]:
```suggestion
// BackKeyClicked implements tea.Cmd. It creates a message indicating 'Esc' has been clicked.
```

**Review by hedhyw — APPROVED** [review]
Thank you!

**Review by hedhyw — APPROVED** [review]

**Pull request merged**